bot/commands/units.js

```diff
@@ -139,6 +139,13 @@ module.exports = {
         boost(unit)
     }
 
+    // Add integer values to be used for damage calculation.
+    unit.iAtt = BigInt(unit.att * 100);
+    unit.iBonus = BigInt(unit.bonus * 10);
+    unit.iDef = BigInt(unit.def * 100 * unit.bonus);
```

Comment:
Shouldn't you multiple by unit.iBonus instead of unit.bonus at this point?

Reply:
Clearly not, because it works

Reply:
Yeah, the problem is that you can't mix Numbers and BigInts. I can write it as:
`BigInt(unit.def * 10 * Number(unit.iBonus));` but it is not much better.

Reply:
Yeah, no changes needed